README.md

```diff
@@ -70,16 +71,55 @@ $ sbt
 
 ```shell
 > projects
-[info] In file:/Users/Juan/Development/hmda-platform/
+[info] In file:/Users/marinj/Development/hmda-platform/
+[info] 	   api
 [info] 	 * hmda
-[info] 	   model
-[info] 	   parser
+[info] 	   modelJS
+[info] 	   modelJVM
+[info] 	   parserJS
+[info] 	   parserJVM
+[info] 	   platformTestJS
+[info] 	   platformTestJVM
 
 > ~re-start
 ```
 
 Confirm that the platform is up and running by browsing to http://localhost:8080
 
+To build the project artifacts, there are two options depending on which runtime is being targeted. 
+
+* To build JVM artifacts (the default, includes all projects), from the sbt prompt:
+
+```shell
+> assembly
+```
+
+This task will create a `fat jar`, which can be executed directly on any JDK8 compliant JVM:
```

Comment:
Is this supposed to be "fat jar"?  It seems like there's a `fat jar` command as it stands.